contracts/fast/FastTokenFacet.sol

```diff
@@ -11,7 +12,7 @@ import './FastHistoryFacet.sol';
 import './lib/AFastFacet.sol';
 
 
-contract FastTokenFacet is AFastFacet, IERC20, IERC1404 {
+contract FastTokenFacet is AFastFacet, IERC20, IERC165, IERC1404 {
```

Comment:
I wouldn't say that the "Token" aspect of a FAST is IERC165 compliant. I would say that its top-level is - eg `FastTopFacet`.

Reply:
Same idea as per why the `FastTokenFacet` isn't `IHasMembers` or `IHasGovernors` compliant - the `FastTopFacet` is.